mesa/experimental/cell_space/grid.py

```diff
@@ -0,0 +1,179 @@
+from random import Random
+
+from mesa.experimental.cell_space import Cell, DiscreteSpace
+
+
+class Grid(DiscreteSpace):
```

Comment:
Is it really this useful to make another layer between `DiscreteSpace` and the user accessible grids?

Also do we want stuff like DiscreteSpace and Grid private (so with an underscore)?

Reply:
Using an underscore in class names is atypical. The better and correct way to declare stuff public is through `__all__`. Which, indeed we could add.

Reply:
We discussed this, and for now exposing everything is fine.